### PR TITLE
Add dependent note for capital investment

### PR DESCRIPTION
**Pull request by JulianRoesner**
# Short Description
This adds a dependent note for capital investment on the success page of the eligibility flow. Because the new text is very close to the one for the cheaper check, the new text replaces the  dependent note of the cheaper check.

# Changes
- Add `data_fits_one_model`
- Add new dependent note to the success page

# Feedback
- Do you see any problems?
- Do you have any ideas for a better name for `data_fits_one_data_model`?

### CODE REVIEW

File: webapp/app/forms/steps/eligibility_steps.py [hunk 1]
```diff
@@ -38,6 +38,14 @@ def data_fits_data_model(data_model, stored_data):
     return True
 
 
+def data_fits_one_data_model(data_models, stored_data):
+    """
+    Method to find out whether the data entered by the user fits at least one model
+    """
+    fits_models = [data_fits_data_model(data_model, stored_data) for data_model in data_models]
+    return any(fits_models)
```
Comment (frederike-ramin) [inline webapp/app/forms/steps/eligibility_steps.py:46]: You asked about a renaming, so maybe `data_fits_data_model_from/in_list` could be possible? But I think `data_fits_one_data_model` is also fine.

File: webapp/app/translations/de/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -957,6 +957,17 @@ msgstr ""
...
+#: app/forms/steps/eligibility_steps.py:748
+msgid "form.eligibility.result-note.capital_investment"
+msgstr ""
+"Die Besteuerung von Kapitalerträgen erledigen Banken und Versicherungen "
+"für Sie. Wenn Sie Freistellungsaufträge erteilt haben, dann erhalten Sie "
+"dort auch Ihren Sparerfreibetrag. Wenn Sie keine Freistellungsaufträge "
+"erteilt haben oder mit der Höhe der für Sie abgeführten Steuern nicht "
+"einverstanden sind, dann können Sie in der Einkommensteuererklärung eine "
+"Änderung der Steuerbelastung beantragen. Der Steuerlotse bietet Ihnen "
+"dafür allerdings derzeit keine Möglichkeiten."
```
Comment (frederike-ramin) [inline webapp/app/translations/de/LC_MESSAGES/messages.po:969]: Why is the cheaper_check message not gone? Is it used anywhere else or did you just not commit the changes to this file?

**Review by frederike-ramin — APPROVED** [review]
Looks good to me. :)

File: webapp/app/translations/de/LC_MESSAGES/messages.po (hunk 2 above)
Comment (JulianRoesner) [reply webapp/app/translations/de/LC_MESSAGES/messages.po:969]: Good catch. I did not commit the changes

**Pull request merged**